slices/php-common.yaml

```diff
@@ -0,0 +1,33 @@
+package: php-common
+
+essential:
+  - php-common_copyright
+
+slices:
+  bins:
+    essential:
+      - base-files_bin
+      - coreutils_bins
+      - dash_bins
+      - findutils_bins
+      - libssl3t64_libs
+      - php-common_libs
+      - psmisc_bins
```

Comment:
Can you identify all the specific utilities from `psmisc` are being used by all the various scripts in `php-common`?
Because it's not immediately apparent what exactly needs it from what I can see?
Could it be that it's only needed within the maintainer scripts for this package and wouldn't be needed for Chisel?
```shell
$ egrep 'fuser|killall|peekfd|prtstat|pslog|pstree' /usr/sbin/php*
$ egrep -r 'fuser|killall|peekfd|prtstat|pslog|pstree' /usr/lib/php
```